### ISSUE TITLE
Newline in extra options for rsync task breaks generated script (Includes fix)

### ISSUE THREAD

**Issue by omv-zoki** [openmediavault/openmediavault#1223]
**Describe the bug**

OMV6 uses a textarea in the configuration of rsync jobs. This allows the user to enter new line characters `\n` which will break the generated `/var/lib/openmediavault/cron.d/rsync-<UUID> job` as the new line characters splits the comand line into multiple.

Example script:
```bash
root@omv-export:/var/lib/openmediavault/cron.d# cat rsync-1ef0422b-72af-4c5c-8859-2dae6d34c8fe
#!/bin/sh
# This file is auto-generated by openmediavault (https://www.openmediavault.org)
# WARNING: Do not edit this file, your changes will get lost.
. /usr/share/openmediavault/scripts/helper-functions

...

touch "/run/rsync-1ef0422b-72af-4c5c-8859-2dae6d34c8fe"
rsync --verbose --log-file="/var/log/rsync.log" --quiet --dry-run --archive --no-perms --delete
--exclude=aquota.group --exclude=aquota.user "/srv/dev-disk-by-uuid-54705666-fb2a-4c5c-ae08-3a36a77186d0/homeexport/" "/srv/dev-disk-by-uuid-54705666-fb2a-4c5c-ae08-3a36a77186d0/netzwerkpfadexport/" & wait $!
exit 0
```

**To Reproduce**

1. Go to Services / rsync / tasks
2. Create a rsync job and hit enter in the extra options text area between two options
3. Save and Apply
4. Check the generated script in /var/lib/openmediavault/cron.d/

**Expected behavior**

a correct script is generated without line breaks inside the rsync command.

**Reference to Forum**

See: https://forum.openmediavault.org/index.php?thread/41969-omv6-backup-full-drive/&postID=303744#post303744

**openmediavault Server:**

- OS version: not relevant
- openmediavault version:  6.0.13-2

**Additional context**

I like the option to enter additional args on multiple lines. To make this work, one would have to pipe the `job.extraoptions` through a filter replacing  \n  by a space char.

just replace 
```
{%- if job.extraoptions | length > 0 %}{{ separator }}{{ job.extraoptions }}{%- endif -%} 
```
by 
```
{%- if job.extraoptions | length > 0 %}{{ separator }}{{ job.extraoptions | regex_replace('[\\r\\n\\t]+',' ') }}{%- endif -%} 
```

Do you need a PR?

**Comment by votdev** [comment]
I think it is much easier to replace the textarea field by a text input field, thus the user can not enter a newline.

**Comment by votdev** [comment]
Fixed in `openmediavault 6.0.14`.
